components/engine/src/mutable.rs

```diff
@@ -1,9 +1,9 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
-use crate::rocks::{CFHandle, Writable};
+use crate::rocks::{CFHandle, Writable, WriteBatchBase};
 use crate::Result;
 
-pub trait Mutable: Writable {
```

Comment:
Could you document what these traits are for please? I can't figure it out, not why they are different.

Reply:
Writable are implemented by DB, it could be write keys by multiple thread concurrently. And Mutable is for `WriteBatch`, it must be updated by only one thread.